modules/weko-index-tree/weko_index_tree/utils.py

```diff
@@ -328,3 +331,52 @@ def get_admin_coverpage_setting():
     except Exception as ex:
         current_app.logger.debug(ex)
     return avail == 'enable'
+
+
+def get_ES_records_data_by_indexes(index_ids):
+    """Get data from elastic search.
+
+    Arguments:
+        start_date {string} -- start date
+        end_date {string} -- end date
+
+    Returns:
+        dictionary -- elastic search data
+
+    """
+    records_search = RecordsSearch()
+    records_search = records_search.with_preference_param().params(version=False)
+    records_search._index[0] = current_app.config['SEARCH_UI_SEARCH_INDEX']
+
+    from weko_search_ui.query import item_search_by_list_index_id
+
+    search_instance, _qs_kwargs = item_search_by_list_index_id(None,
+                                                               records_search,
+                                                               index_ids)
+    search_result = search_instance.execute()
+    rd = search_result.to_dict()
+    return rd
+
+def generate_path(index_ids):
+    """Get data from elastic search.
+
+    Arguments:
+        start_date {string} -- start date
+        end_date {string} -- end date
+
+    Returns:
+        dictionary -- elastic search data
+
+    """
+    current_app.logger.debug(index_ids)
```

Comment:
Does   the necessary information be output to the log     If you catch an error, you need to keep a log that can identify the cause   of the error.

delete it

Reply:
Thank, It had been fixed.